task/09-functions-n-closures-tasks.js

```diff
@@ -104,7 +108,16 @@ function memoize(func) {
  * retryer() => 2
  */
 function retry(func, attempts) {
-    throw new Error('Not implemented');
+    return () => {
+        let attemptsCounter = 0;
+        while(true) {
```

Comment:
generally you don't want to use infinite loop if the loop is finite. In this case it has n iterations - the best loop for this kind of thing is  for-loop